pkg/testcase/secretsencrypt.go

```diff
@@ -0,0 +1,170 @@
+package testcase
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestSecretsEncryption() {
+	nodes, errGetNodes := shared.GetNodesByRoles("etcd", "control-plane")
+	Expect(nodes).NotTo(BeEmpty())
+	Expect(errGetNodes).NotTo(HaveOccurred(), "error getting etcd/control-plane nodes")
+
+	product, err := shared.Product()
+	Expect(err).NotTo(HaveOccurred(), "error getting product from config")
+
+	ips := getNodeIps(nodes)
+
+	errSecret := shared.CreateSecret("secret1", "default")
```

Comment:
can we have break line here ? separating code block concerns 
```

	errSecret := shared.CreateSecret("secret1", "default")
	Expect(errSecret).NotTo(HaveOccurred(), "error creating secret")

	shared.LogLevel("INFO", "TEST: 'CLASSIC' Secrets Encryption method")

	index := len(nodes) - 1
	secretsEncryptOps("prepare", product, nodes[index].ExternalIP, ips)
	secretsEncryptOps("rotate", product, nodes[index].ExternalIP, ips)
	secretsEncryptOps("reencrypt", product, nodes[index].ExternalIP, ips)

```

Reply:
fixed